lua/compe/init.lua

```diff
@@ -38,7 +38,7 @@ local compe = {}
 compe.helper = Helper
 
 --- setup
-compe.setup = function(config)
+compe.setup = function(config, bufnr)
   Pattern.set_filetype_config('vim', {
```

Comment:
`Pattern.set_filetype_config` can be potentially called multiple times. Is this a problem?

Reply:
Yes. We should move it another place.

Reply:
Maybe on initialization, in `plugin/compe.vim`?

Reply:
I think the top of `init.lua` would be good.